go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -284,6 +299,59 @@ var DescribeTableAsOfScriptTest = queries.ScriptTest{
 	},
 }
 
+var DescribeTableAsOfWithFkScriptTest = queries.ScriptTest{
+	Name: "Describe table as of with foreign key constraints",
```

Comment:
Where are the foreign key constraints in this test?

Reply:
Wrong tests added :(
These are reverted and correct tests are in place. Sorry about that.